hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/exec/utils/FrameUtilsTest.java

```diff
@@ -36,7 +37,7 @@
 
 class FrameUtilsTest {
     private static final Set<Class<?>> toBeTested =
-            new HashSet<>(Arrays.asList(FrameUtils.class, ConversionUtils.class, OpUtils.class));
+            new HashSet<>(Arrays.asList(FrameUtils.class, ConversionUtils.class, OpUtils.class, OpcodeUtils.class));
```

Comment:
(Pretty hard core testing that your utility classes enforce not creating instances!  Even I, a total believer in unit tests, wouldn't have gone that far ...)